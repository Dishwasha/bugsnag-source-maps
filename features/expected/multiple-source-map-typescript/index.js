"use strict";
console.log('hi from index');
//# sourceMappingURL=index.js.map
