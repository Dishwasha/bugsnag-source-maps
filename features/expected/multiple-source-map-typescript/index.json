{
    "version": 3,
    "file": "index.js",
    "sourceRoot": "",
    "sources": [
        "src/index.ts"
    ],
    "names": [],
    "mappings": ";AAAA,OAAO,CAAC,GAAG,CAAC,eAAe,CAAC,CAAA",
    "sourcesContent": [
        "console.log('hi from index')\n"
    ]
}
